conda/multisheller/wb-toolbox_activate.msh

```diff
@@ -1,7 +1,15 @@
+# For some reason appending two times the same variable inside the same if does not work in command prompt
```

Comment:
In my opinion, this sentence would be clearer if we change it like in the following:
```
For some reason, appending two times the same variable inside the same "if" statement does not work in command prompt.
```